src/server.cc

```diff
@@ -244,8 +244,8 @@ InferenceServer::Init()
   }
 
 #endif  // TRITON_ENABLE_GPU
-
-  status = EnablePeerAccess(min_supported_compute_capability_);
+  status =
+      EnablePeerAccess(min_supported_compute_capability_, enable_peer_access_);
   if (!status.IsOk()) {
     // failed to enable peer access is not critical, just inefficient.
     LOG_WARNING << status.Message();
```

Comment:
if the user has explicitly asked for enabling peer access - should we error instead of warn?

Reply:
We can do that, but I feel as this warning has existed till now changing to ERROR can cause existing customer scripts (if any) and/or our tests to maybe break?

Reply:
> if the user has explicitly asked for enabling peer access - should we error instead of warn?

Default is on for performance reason. I don't know about error out since it doesn't break functionality.